error-codes.md

```diff
@@ -484,6 +514,42 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
     Check the state/monitor/log of the TiFlash server.
 
+### MySQL native error messages
+
+* ERROR 2013 (HY000): Lost connection to MySQL server during query
+
+    The complete error message: `ERROR 2013 (HY000): Lost connection to MySQL server during query`
+
+    You can handle the error in the following ways:
+
+    - Check whether panic is in the log.
+    - Check whether OOM exists in dmesg using `dmesg -T | grep -i oom`.
+    - A long time of no access might also lead to this error. It is usually caused by TCP timeout. If TCP is not used for a long time, the operating system kills it.
+
+* ERROR 1105 (HY000): other error: unknown error Wire Error(InvalidEnumValue(4004))
+
+    The complete error message: `ERROR 1105 (HY000): other error: unknown error Wire Error(InvalidEnumValue(4004))`
+
+    This error usually occurs when the version of TiDB does not match with the version of TiKV. To avoid version mismatch, upgrade all components when you upgrade the version.
+
+* ERROR 1148 (42000): the used command is not allowed with this TiDB version
+
+    The complete error message: `ERROR 1148 (42000): the used command is not allowed with this TiDB version`
+
+    When you execute the `LOAD DATA LOCAL` statement but the MySQL client does not allow executing this statement (the value of the `local_infile` option is 0), this error occurs.
+
+    The solution is to use the `--local-infile=1` option when you start the MySQL client. For example, use command like `mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000`. The default value of `local-infile` is different in different versions of MySQL client, therefore you need to configure it in some MySQL clients and do not need to configure it in some others.
+
+* ERROR 9001 (HY000): PD server timeout start timestamp may fall behind safe point
+
+    The complete error message: `ERROR 9001 (HY000): PD server timeout start timestamp may fall behind safe point`
+
+    This error occurs when TiDB fails to access PD. A worker in the TiDB background continuously queries the safepoint from PD and this error occurs if it fails to query within 100s. Generally, it is because the disk on PD is slow and busy or the network failed between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](/error-codes.md).
```

Comment:
```suggestion
    This error occurs when TiDB fails to access PD. A worker in the TiDB background continuously queries the safepoint from PD and reports this error if it fails to query within 100s. Generally, it is because the disk on PD is slow and busy or the network failed between TiDB and PD. For the details of common errors, see [Error Number and Fault Diagnosis](/error-codes.md).
```